### DISCUSSION TITLE
Close tab button

### DISCUSSION THREAD

**Discussion by MonkAlex** [piroor/treestyletab#3204] [Q&A] [answered]
Two day ago i lost close button on tabs. Sometimes i see that behavior and just restart panel (f1) or browser.

![img](https://user-images.githubusercontent.com/6784764/190892676-09b8b9c0-0298-4365-9414-2b54311675e3.png)

I have button without any image\symbol, button have tooltip.

UPD: tst version 3.9.2, not use custom styling.
UPD2: expand button have the same behavior.

**Comment by piroor** [comment, marked as answer]
Already filed as an issue: #3142

And there is a related bug of Firefox side: [1763420 - Sidebar images linked in css are not always rendered](https://bugzilla.mozilla.org/show_bug.cgi?id=1763420)
